e2e/create-waku.spec.ts

```diff
@@ -43,7 +43,10 @@ test('should create waku with default setup work', async () => {
   expect(files).toContain('package.json');
   expect(files).toContain('src');
   expect(files).toContain('tsconfig.json');
-  await terminate(childProcess.pid!);
+  exec(`rm -rf ${cwd}`);
+  // Why is this function needed?
+  // Can not terminate sub process automatically when tests are finished?
+  // await terminate(childProcess.pid);
```

Comment:
@himself65

Could you explain why `terminate` is needed?

Reply:
If adding this line, the error occurs below:

```
[[chromium] › create-waku.spec.ts:8:1 › should create waku with default setup work: e2e/create-waku.spec.ts#L1](https://github.com/dai-shi/waku/pull/808/files#annotation_26269905744)
  1) [chromium] › create-waku.spec.ts:8:1 › should create waku with default setup work ─────────────
    Error: kill ESRCH

        at handlePsTreeCallback (/Users/runner/work/waku/waku/node_modules/.pnpm/terminate@2.8.0/node_modules/terminate/handlePsTreeCallback.js:28:13)
        at /Users/runner/work/waku/waku/node_modules/.pnpm/terminate@2.8.0/node_modules/terminate/terminate.js:44:5
        at /Users/runner/work/waku/waku/node_modules/.pnpm/ps-tree@1.2.0/node_modules/ps-tree/index.js:87:7
        at Stream.a.end (/Users/runner/work/waku/waku/node_modules/.pnpm/event-stream@3.3.4/node_modules/event-stream/index.js:87:5)
        at end (/Users/runner/work/waku/waku/node_modules/.pnpm/map-stream@0.1.0/node_modules/map-stream/index.js:116:39)
        at Stream.stream.end (/Users/runner/work/waku/waku/node_modules/.pnpm/map-stream@0.1.0/node_modules/map-stream/index.js:122:5)
        at drain (/Users/runner/work/waku/waku/node_modules/.pnpm/through@2.3.8/node_modules/through/index.js:34:23)
        at Stream.stream.queue.stream.push (/Users/runner/work/waku/waku/node_modules/.pnpm/through@2.3.8/node_modules/through/index.js:45:5)
        at Stream.<anonymous> (/Users/runner/work/waku/waku/node_modules/.pnpm/split@0.3.3/node_modules/split/index.js:60:10)
        at _end (/Users/runner/work/waku/waku/node_modules/.pnpm/through@2.3.8/node_modules/through/index.js:65:9)
        at Stream.stream.end (/Users/runner/work/waku/waku/node_modules/.pnpm/through@2.3.8/node_modules/through/index.js:74:5)
```

Reply:
terminate will stop all the child process realted to the pid, I think the issues was I cannot stop the cp by `cp.kill()` and wait for exit for some reason

Reply:
If it's confirmed that it's not necessary, remove it entirely L47-49.

Reply:
Yeah It is not necessary. 

Check this commit! 1c134df